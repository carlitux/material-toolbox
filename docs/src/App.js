// @flow
/* eslint-disable */
import * as React from 'react';
import { Link, Route } from 'react-router-dom';

import { LayoutGrid, LayoutInner, LayoutCell } from 'material-toolbox/layout-grid';
import { PermanentDrawer, DrawerContent } from 'material-toolbox/drawer';
import { List, ListItem } from 'material-toolbox/list';
// import Ripple from 'material-toolbox/ripple';
import Theme from 'material-toolbox/theme';
import {
  ToolbarTitle,
  ToolbarIcon,
  ToolbarRow,
  ToolbarSection,
  ToolbarFixedAdjust,
  Toolbar,
} from 'material-toolbox/toolbar';
/* eslint-enable */

import HomePage from './HomePage';
import AnimationPage from './AnimationPage';
import TypographyPage from './TypographyPage';
import ThemePage from './ThemePage';
import Hero from './Hero';
import ListsPage from './ListsPage';
import LayoutGridPage from './LayoutGridPage';
import RipplesPage from './RipplesPage';
import ElevationPage from './ElevationPage';
import MainNav from './MainNav';
import DrawerPage from './DrawerPage';
import ButtonPage from './ButtonPage';
import FABPage from './FABPage';
import IconTogglePage from './IconTogglePage';
import styles from './styles.scss';

type State = {
  adjustFixedStyle: { [string]: any },
  titleStyle: { [string]: any },
};

export default class App extends React.Component<{}, State> {
  state = {
    adjustFixedStyle: {},
    titleStyle: {},
  };

  handleUpdateFixedToolbar = (property: string, value: number) => {
    this.setState({
      adjustFixedStyle: { ...this.state.adjustFixedStyle, [property]: value },
    });
  };

  handleUpdateTitleStyle = (property: string, value: number) => {
    this.setState({
      titleStyle: { ...this.state.titleStyle, [property]: value },
    });
  };

  render() {
    return (
      <div className={styles['demo-body']}>
        <Toolbar
          fixed
          onUpdateTitleStyle={this.handleUpdateTitleStyle}
          onUpdateFixedToolbar={this.handleUpdateFixedToolbar}>
          <ToolbarRow>
            <ToolbarSection align="start">
              <ToolbarTitle
                label="React Material Component Web"
                style={this.state.titleStyle}
              />
            </ToolbarSection>
          </ToolbarRow>
        </Toolbar>

        <ToolbarFixedAdjust>
          <main className="demo-content">
            <PermanentDrawer>
              <DrawerContent>
                <MainNav />
              </DrawerContent>
            </PermanentDrawer>
            <div>
              <Route exact path="/" component={Hero} />
              <LayoutGrid>
                <LayoutInner>
                  <Route exact path="/" component={HomePage} />
                  <Route path="/animation" component={AnimationPage} />
                  <Route path="/typography" component={TypographyPage} />
                  <Route path="/theme" component={ThemePage} />
                  <Route path="/lists" component={ListsPage} />
                  <Route path="/layout-grid" component={LayoutGridPage} />
                  <Route path="/ripples" component={RipplesPage} />
                  <Route path="/elevation" component={ElevationPage} />
                  <Route path="/drawers" component={DrawerPage} />
                  <Route exact path="/buttons" component={ButtonPage} />
                  <Route path="/buttons/floating" component={FABPage} />
                  <Route path="/buttons/icons" component={IconTogglePage} />
                </LayoutInner>
              </LayoutGrid>
            </div>
          </main>
        </ToolbarFixedAdjust>
      </div>
    );
  }
}
