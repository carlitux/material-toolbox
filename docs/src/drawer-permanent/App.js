// @flow
/* eslint-disable */
import React from 'react';

import { Text } from 'material-toolbox/typography';
import Elevation from 'material-toolbox/elevation';
import { PermanentDrawer, DrawerContent } from 'material-toolbox/drawer';
import { ListGroup, List, ListItem, ListDetail } from 'material-toolbox/list';
import { Toolbar, ToolbarIcon, ToolbarSection, ToolbarTitle, ToolbarRow, ToolbarFixedAdjust } from 'material-toolbox/toolbar';
/* eslint-enable */
import './styles.scss';

type State = {
  wide: boolean,
  tall: boolean,
};

export default class PermanentDrawerApp extends React.Component<{}, State> {
  state = {
    wide: false,
    tall: false,
  };

  toogleWide = () => {
    this.setState(state => ({
      wide: !state.wide,
    }));
  };

  toogleTall = () => {
    this.setState(state => ({
      tall: !state.tall,
    }));
  };

  render() {
    return (
      <div>
        <Toolbar fixed>
          <ToolbarRow>
            <ToolbarSection align="start">
              <ToolbarIcon icon="arrow_back" isMenu />
              <ToolbarTitle label="Permanent Drawer Below Toolbar" />
            </ToolbarSection>
          </ToolbarRow>
        </Toolbar>
        <ToolbarFixedAdjust>
          <main className="demo-content">
            <PermanentDrawer>
              <DrawerContent>
                <ListGroup>
                  <List component="nav">
                    <ListItem
                      component="a"
                      href="#"
                      className="mdc-permanent-drawer--selected">
                      <ListDetail className="material-icons" component="i">
                        inbox
                      </ListDetail>
                      Inbox
                    </ListItem>
                    <ListItem component="a" href="#">
                      <ListDetail className="material-icons" component="i">
                        start
                      </ListDetail>
                      Start
                    </ListItem>
                    <ListItem component="a" href="#">
                      <ListDetail className="material-icons" component="i">
                        send
                      </ListDetail>
                      Send Mail
                    </ListItem>
                    <ListItem component="a" href="#">
                      <ListDetail className="material-icons" component="i">
                        drafts
                      </ListDetail>
                      Drafts
                    </ListItem>
                  </List>
                  <ListItem devider component="hr" />
                  <List component="nav">
                    <ListItem component="a" href="#">
                      <ListDetail className="material-icons" component="i">
                        mail
                      </ListDetail>
                      All Mail
                    </ListItem>
                    <ListItem component="a" href="#">
                      <ListDetail className="material-icons" component="i">
                        delete
                      </ListDetail>
                      Trash
                    </ListItem>
                    <ListItem component="a" href="#">
                      <ListDetail className="material-icons" component="i">
                        report
                      </ListDetail>
                      Spam
                    </ListItem>
                  </List>
                </ListGroup>
              </DrawerContent>
            </PermanentDrawer>
            <div className="demo-main">
              <Text component="h1" textStyle="display1">
                Permanent Drawer
              </Text>
              <Text component="p" textStyle="body1">
                It sits to the left of this content.
              </Text>
              <div className="extra-content-wrapper">
                <button onClick={this.toogleWide}>
                  Toggle extra-tall content
                </button>
                {this.state.wide && (
                  <Elevation elevation={2}>
                    <div id="extra-wide-content">&nbsp;</div>
                  </Elevation>
                )}
              </div>
              <div className="extra-content-wrapper">
                <button onClick={this.toogleTall}>
                  Toggle extra-tall content
                </button>
                {this.state.tall && (
                  <Elevation elevation={2}>
                    <div id="extra-tall-content">&nbsp;</div>
                  </Elevation>
                )}
              </div>
            </div>
          </main>
        </ToolbarFixedAdjust>
      </div>
    );
  }
}
