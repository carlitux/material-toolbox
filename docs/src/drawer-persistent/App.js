// @flow
/* eslint-disable */
import React from 'react';

import { Text } from 'material-toolbox/typography';
import Elevation from 'material-toolbox/elevation';
import { PersistentDrawer, DrawerContent, DrawerToolbarSpacer } from 'material-toolbox/drawer';
import { ListGroup, List, ListItem, ListDetail } from 'material-toolbox/list';
import { Toolbar, ToolbarIcon, ToolbarSection, ToolbarTitle, ToolbarRow, ToolbarFixedAdjust } from 'material-toolbox/toolbar';
/* eslint-enable */
import './styles.scss';

type State = {
  isDrawerOpen: boolean,
};

export default class PermanentDrawerApp extends React.Component<{}, State> {
  state = {
    isDrawerOpen: false,
  };

  toogleDrawer = () => {
    this.setState(state => ({
      isDrawerOpen: !state.isDrawerOpen,
    }));
  };

  handleOpen = () => {
    this.setState(state => ({
      isDrawerOpen: true,
    }));
  };

  handleClose = () => {
    this.setState(state => ({
      isDrawerOpen: false,
    }));
  };

  render() {
    return [
      <PersistentDrawer
        key="drawer"
        open={this.state.isDrawerOpen}
        onOpen={this.handleOpen}
        onClose={this.handleClose}>
        <DrawerToolbarSpacer />
        <DrawerContent>
          <ListGroup>
            <List component="nav">
              <ListItem
                component="a"
                href="#"
                className="mdc-persistent-drawer--selected">
                <ListDetail className="material-icons" component="i">
                  inbox
                </ListDetail>
                Inbox
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  start
                </ListDetail>
                Start
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  send
                </ListDetail>
                Send Mail
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  drafts
                </ListDetail>
                Drafts
              </ListItem>
            </List>
            <ListItem devider component="hr" />
            <List component="nav">
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  mail
                </ListDetail>
                All Mail
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  delete
                </ListDetail>
                Trash
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  report
                </ListDetail>
                Spam
              </ListItem>
            </List>
          </ListGroup>
        </DrawerContent>
      </PersistentDrawer>,

      <div key="main" className="demo-content">
        <Toolbar>
          <ToolbarRow>
            <ToolbarSection align="start">
              <ToolbarIcon icon="menu" isMenu onClick={this.toogleDrawer} />
              <ToolbarTitle label="Persistent Drawer" />
            </ToolbarSection>
          </ToolbarRow>
        </Toolbar>

        <main className="demo-content">
          <div className="demo-main">
            <Text component="h1" textStyle="display1">
              Persistent Drawer
            </Text>
            <Text component="p" textStyle="body1">
              Click the menu icon above to open and close the drawer.
            </Text>
          </div>
        </main>
      </div>,
    ];
  }
}
