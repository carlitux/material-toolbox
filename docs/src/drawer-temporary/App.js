// @flow
/* eslint-disable */
import React from 'react';

import { Text } from 'material-toolbox/typography';
import Elevation from 'material-toolbox/elevation';
import Theme from 'material-toolbox/theme';
import { TemporaryDrawer, DrawerContent, DrawerHeader } from 'material-toolbox/drawer';
import { ListGroup, List, ListItem, ListDetail } from 'material-toolbox/list';
import { Toolbar, ToolbarIcon, ToolbarSection, ToolbarTitle, ToolbarRow, ToolbarFixedAdjust } from 'material-toolbox/toolbar';
/* eslint-enable */
import './styles.scss';

type State = {
  isDrawerOpen: boolean,
};

export default class PermanentDrawerApp extends React.Component<{}, State> {
  state = {
    isDrawerOpen: false,
  };

  toggleDrawer = () => {
    this.setState(state => ({
      isDrawerOpen: !state.isDrawerOpen,
    }));
  };

  handleOpen = () => {
    this.setState(state => ({
      isDrawerOpen: true,
    }));
  };

  handleClose = () => {
    this.setState(state => ({
      isDrawerOpen: false,
    }));
  };

  render() {
    return [
      <Toolbar key="toolbar" fixed>
        <ToolbarRow>
          <ToolbarSection align="start">
            <ToolbarIcon icon="menu" isMenu onClick={this.toggleDrawer} />
            <ToolbarTitle label="Temporary Drawer" />
          </ToolbarSection>
        </ToolbarRow>
      </Toolbar>,
      <TemporaryDrawer
        key="drawer"
        open={this.state.isDrawerOpen}
        onOpen={this.handleOpen}
        onClose={this.handleClose}>
        <Theme theme="primary" textOn="primary" textStyle="primary">
          <DrawerHeader>Header here</DrawerHeader>
        </Theme>
        <DrawerContent>
          <ListGroup>
            <List component="nav">
              <ListItem
                component="a"
                href="#"
                className="mdc-persistent-drawer--selected">
                <ListDetail className="material-icons" component="i">
                  inbox
                </ListDetail>
                Inbox
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  start
                </ListDetail>
                Start
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  send
                </ListDetail>
                Send Mail
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  drafts
                </ListDetail>
                Drafts
              </ListItem>
            </List>
            <ListItem devider component="hr" />
            <List component="nav">
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  mail
                </ListDetail>
                All Mail
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  delete
                </ListDetail>
                Trash
              </ListItem>
              <ListItem component="a" href="#">
                <ListDetail className="material-icons" component="i">
                  report
                </ListDetail>
                Spam
              </ListItem>
            </List>
          </ListGroup>
        </DrawerContent>
      </TemporaryDrawer>,

      <ToolbarFixedAdjust key="main">
        <main className="demo-main">
          <Text component="h1" textStyle="display1">
            Temporary Drawer
          </Text>
          <Text component="p" textStyle="body1">
            Click the menu icon above to open and close the drawer.
          </Text>
        </main>
      </ToolbarFixedAdjust>,
    ];
  }
}
