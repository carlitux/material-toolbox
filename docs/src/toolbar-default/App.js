// @flow
/* eslint-disable */
import React from 'react';

import { Text } from 'material-toolbox/typography';
import {
  Toolbar,
  ToolbarIcon,
  ToolbarSection,
  ToolbarTitle,
  ToolbarRow,
  ToolbarFixedAdjust,
} from 'material-toolbox/toolbar';
/* eslint-enable */
import './styles.scss';

// eslint-disable-next-line
export default class DefaultToolbarApp extends React.Component<{}> {
  render() {
    return (
      <div>
        <Toolbar>
          <ToolbarRow>
            <ToolbarSection align="start">
              <ToolbarIcon className="material-icons" isMenu>
                menu
              </ToolbarIcon>
              <ToolbarTitle label="Title" />
            </ToolbarSection>
            <ToolbarSection align="end">
              <ToolbarIcon
                className="material-icons"
                aria-label="Download"
                alt="Download">
                file_download
              </ToolbarIcon>
              <ToolbarIcon
                className="material-icons"
                aria-label="Print this page"
                alt="Print this page">
                print
              </ToolbarIcon>
              <ToolbarIcon
                className="material-icons"
                aria-label="Bookmark this page"
                alt="Bookmark this page">
                bookmark
              </ToolbarIcon>
            </ToolbarSection>
          </ToolbarRow>
        </Toolbar>
        <main className="demo-content">
          <Text component="div" textStyle="body1">
            <p className="demo-paragraph">
              Pellentesque habitant morbi tristique senectus et netus et
              malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat
              vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
              amet quam egestas semper. Aenean ultricies mi vitae est.
              Pellentesque habitant morbi tristique senectus et netus et
              malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat
              vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
              amet quam egestas semper. Aenean ultricies mi vitae est.
            </p>
            <p className="demo-paragraph">
              Pellentesque habitant morbi tristique senectus et netus et
              malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat
              vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
              amet quam egestas semper. Aenean ultricies mi vitae est.
              Pellentesque habitant morbi tristique senectus et netus et
              malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat
              vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
              amet quam egestas semper. Aenean ultricies mi vitae est.
            </p>
            <p className="demo-paragraph">
              Pellentesque habitant morbi tristique senectus et netus et
              malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat
              vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
              amet quam egestas semper. Aenean ultricies mi vitae est.
              Pellentesque habitant morbi tristique senectus et netus et
              malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat
              vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit
              amet quam egestas semper. Aenean ultricies mi vitae est.
            </p>
          </Text>
        </main>
      </div>
    );
  }
}
