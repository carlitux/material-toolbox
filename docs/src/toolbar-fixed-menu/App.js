// @flow
/* eslint-disable */
import React from 'react';

import { Text } from 'material-toolbox/typography';
import { SimpleMenu, SimpleMenuItem } from 'material-toolbox/menu/simple';
import {
  Toolbar,
  ToolbarIcon,
  ToolbarSection,
  ToolbarTitle,
  ToolbarRow,
  ToolbarFixedAdjust,
} from 'material-toolbox/toolbar';
/* eslint-enable */
import './styles.scss';

type State = {
  show: boolean,
};

// eslint-disable-next-line
export default class FixedToolbarApp extends React.Component<{}, State> {
  state = {
    show: false,
  };

  handleCancel = () => {
    this.setState({ show: false });
  };

  handleClick = () => {
    this.setState({ show: true });
  };

  render() {
    return (
      <div>
        <Toolbar fixed>
          <ToolbarRow>
            <ToolbarSection align="start">
              <ToolbarIcon className="material-icons" isMenu>
                menu
              </ToolbarIcon>
              <ToolbarTitle label="Title" />
            </ToolbarSection>
            <ToolbarSection align="end">
              <ToolbarIcon
                className="material-icons"
                aria-label="Download"
                alt="Download">
                file_download
              </ToolbarIcon>
              <ToolbarIcon
                className="material-icons"
                aria-label="Print this page"
                alt="Print this page">
                print
              </ToolbarIcon>
              <ToolbarIcon
                className="material-icons"
                aria-label="More"
                alt="More"
                onClick={this.handleClick}>
                more_vert
              </ToolbarIcon>
              <div className="mdc-menu-anchor">
                <SimpleMenu
                  onSelect={() => {}}
                  onCancel={this.handleCancel}
                  show={this.state.show}>
                  <SimpleMenuItem>Back</SimpleMenuItem>
                  <SimpleMenuItem>Forward</SimpleMenuItem>
                  <SimpleMenuItem>Reload</SimpleMenuItem>
                  <SimpleMenuItem devider />
                  <SimpleMenuItem>Save As</SimpleMenuItem>
                </SimpleMenu>
              </div>
            </ToolbarSection>
          </ToolbarRow>
        </Toolbar>
        <ToolbarFixedAdjust>
          <main className="demo-content">
            <Text component="div" textStyle="body1">
              <p className="demo-paragraph">
                Pellentesque habitant morbi tristique senectus et netus et
                malesuada fames ac turpis egestas. Vestibulum tortor quam,
                feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu
                libero sit amet quam egestas semper. Aenean ultricies mi vitae
                est. Pellentesque habitant morbi tristique senectus et netus et
                malesuada fames ac turpis egestas. Vestibulum tortor quam,
                feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu
                libero sit amet quam egestas semper. Aenean ultricies mi vitae
                est.
              </p>
              <p className="demo-paragraph">
                Pellentesque habitant morbi tristique senectus et netus et
                malesuada fames ac turpis egestas. Vestibulum tortor quam,
                feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu
                libero sit amet quam egestas semper. Aenean ultricies mi vitae
                est. Pellentesque habitant morbi tristique senectus et netus et
                malesuada fames ac turpis egestas. Vestibulum tortor quam,
                feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu
                libero sit amet quam egestas semper. Aenean ultricies mi vitae
                est.
              </p>
              <p className="demo-paragraph">
                Pellentesque habitant morbi tristique senectus et netus et
                malesuada fames ac turpis egestas. Vestibulum tortor quam,
                feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu
                libero sit amet quam egestas semper. Aenean ultricies mi vitae
                est. Pellentesque habitant morbi tristique senectus et netus et
                malesuada fames ac turpis egestas. Vestibulum tortor quam,
                feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu
                libero sit amet quam egestas semper. Aenean ultricies mi vitae
                est.
              </p>
            </Text>
          </main>
        </ToolbarFixedAdjust>
      </div>
    );
  }
}
