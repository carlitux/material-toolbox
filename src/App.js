// @flow
import * as React from 'react';

import Toolbar from './material-toolbox/toolbar/Toolbar';
import ToolbarRow from './material-toolbox/toolbar/Row';
import ToolbarSection from './material-toolbox/toolbar/Section';
import ToolbarTitle from './material-toolbox/toolbar/Title';
import ToolbarFixedAdjust from './material-toolbox/toolbar/FixedAdjust';

import PermanentDrawer from './material-toolbox/drawer/Permanent';
import DrawerContent from './material-toolbox/drawer/Content';

import MainNav from './MainNav';
import Content from './Content';

export default class App extends React.Component<{}> {
  componentDidMount() {}

  render() {
    return (
      <div className="layout">
        <Toolbar fixed>
          <ToolbarRow>
            <ToolbarSection align="start">
              <ToolbarTitle label="Material Component Web for React - Material Toolbox" />
            </ToolbarSection>
          </ToolbarRow>
        </Toolbar>

        <ToolbarFixedAdjust>
          <main className="main">
            <PermanentDrawer>
              <DrawerContent>
                <MainNav />
              </DrawerContent>
            </PermanentDrawer>
            <div>
              <Content />
            </div>
          </main>
        </ToolbarFixedAdjust>
      </div>
    );
  }
}

//        <ToolbarFixedAdjust>
//          <main className="demo-content">
//            <div>
//              <Route exact path="/" component={Hero} />
//              <LayoutGrid>
//                <LayoutInner>
//                  <Route path="/animation" component={AnimationPage} />
//                  <Route path="/typography" component={TypographyPage} />
//                  <Route path="/theme" component={ThemePage} />
//                  <Route path="/lists" component={ListsPage} />
//                  <Route path="/layout-grid" component={LayoutGridPage} />
//                  <Route path="/ripples" component={RipplesPage} />
//                  <Route path="/elevation" component={ElevationPage} />
//                  <Route path="/drawers" component={DrawerPage} />
//                  <Route exact path="/buttons" component={ButtonPage} />
//                  <Route path="/buttons/floating" component={FABPage} />
//                  <Route path="/buttons/icons" component={IconTogglePage} />
//                  <Route path="/cards" component={CardPage} />
//                  <Route path="/dialogs" component={DialogPage} />
//                  <Route path="/grid-lists" component={GridListPage} />
//                  <Route path="/rtl" component={RTLPage} />
//                  <Route path="/menus" component={MenuPage} />
//                  <Route path="/toolbars" component={ToolbarPage} />
//                  <Route path="/snackbars" component={SnackbarPage} />
//                  <Route
//                    path="/inputs-controls/select-menus"
//                    component={SelectPage}
//                  />
//                  <Route
//                    path="/inputs-controls/sliders"
//                    component={SliderPage}
//                  />
//                  <Route
//                    path="/inputs-controls/switches"
//                    component={SwitchPage}
//                  />
//                  <Route
//                    path="/inputs-controls/text-fields"
//                    component={TextFieldPage}
//                  />
//                  <Route
//                    path="/linear-progress"
//                    component={LinearProgressPage}
//                  />
//                </LayoutInner>
//              </LayoutGrid>
//            </div>
//          </main>
//        </ToolbarFixedAdjust>
