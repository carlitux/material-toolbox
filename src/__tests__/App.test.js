import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('component::App', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
