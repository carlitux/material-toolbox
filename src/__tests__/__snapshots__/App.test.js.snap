// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::App Should match snapshoot 1`] = `
<div
  className="layout"
>
  <Toolbar
    fixed={true}
  >
    <ToolbarRow>
      <ToolbarSection
        align="start"
        shrink={false}
      >
        <ToolbarTitle
          component="span"
          label="Material Component Web for React - Material Toolbox"
        />
      </ToolbarSection>
    </ToolbarRow>
  </Toolbar>
  <ToolbarFixedAdjust>
    <main
      className="main"
    >
      <PermanentDrawer>
        <DrawerContent>
          <MainNav />
        </DrawerContent>
      </PermanentDrawer>
      <div>
        <Content />
      </div>
    </main>
  </ToolbarFixedAdjust>
</div>
`;
